exporters/mongodb3/e2e/e2e_spec.yml

```diff
@@ -34,6 +34,17 @@ scenarios:
         - source: "mongodb.yml"
           except_entities:
             - MONGODB_INSTANCE
+          except_metrics:
+            - mongodb_collstats_storageStats_totalSize
+            - mongodb_top_insert_count
+            - mongodb_top_getmore_count
+            - mongodb_top_queries_count
+            - mongodb_top_readLock_count
+            - mongodb_top_writeLock_count
+            - mongodb_top_remove_count
+            - mongodb_top_update_count
+            - mongodb_dbstats_collections
+            - mongodb_dbstats_views
```

Comment:
These metrics aren't reported in this particular scenario, do you have more details which could be described in a comment here?

Reply:
not really , there were in the exception list already, I couldn't find anything related to this in the API ref